Website/Packages/datax-pipeline/src/modules/flowDefinition/components/flowDefinitionPanel.jsx

```diff
@@ -805,15 +921,23 @@ const mapDispatchToProps = dispatch => ({
     onUpdateInputMode: mode => dispatch(Actions.updateInputMode(mode)),
     onUpdateInputType: type => dispatch(Actions.updateInputType(type)),
     onUpdateInputHubName: name => dispatch(Actions.updateInputHubName(name)),
+    onUpdateInputPath: path => dispatch(Actions.updateInputPath(path)),
     onUpdateInputHubConnection: connection => dispatch(Actions.updateInputHubConnection(connection)),
     onUpdateInputSubscriptionId: id => dispatch(Actions.updateInputSubscriptionId(id)),
     onUpdateInputResourceGroup: name => dispatch(Actions.updateInputResourceGroup(name)),
+    
+    onupdateBatchInputConnection: connection => dispatch(Actions.updateBatchInputConnection(connection)),
```

Comment:
typo: capitalize the U in onUpdateBatchInputConnection to be consistent with other methods you have defined below and in this file.

Reply:
fixed.